plat/mediatek/mt6795/drivers/uart/uart8250.h

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
This is identical to plat/mediatek/mt8173/drivers/uart/uart8250.h. Can it be re-used?